kentik_api_library/pyproject.toml

```diff
@@ -26,6 +26,20 @@ testpaths = ["tests/integration", "tests/unit"]
 [tool.pylint.master]
 fail-under=8.5
 
+[flake8]
+    max-line-length = 120
+    extend-ignore = """
+        E203,
```

Comment:
Good practice: when you don't know how to sort something then sort it alphabetically. It is more maintainable that way

Reply:
Done